test.env

```diff
@@ -81,3 +81,22 @@
 # export SCALE_DB_STORAGES="1Gi 2Gi"
 # export SCALE_CPU_REQUESTS_LIMITS=": 1:1"
 # export SCALE_MEMORY_REQUESTS_LIMITS=": 1Gi:1Gi"
+
+export DURATION=15m
+export PRE_LOAD_DB=true
+export SCALE_ACTIVE_USERS_SPAWN_RATES='100:5'
+export SCALE_BS_USERS_GROUPS='100:20'
+export SCALE_CATALOG_SIZES='1:1 5000:5000 10000:10000 15000:15000 20000:20000 25000:25000 30000:30000'
+export SCALE_CPU_REQUESTS_LIMITS=:
+export SCALE_DB_STORAGES='20Gi'
+export SCALE_MEMORY_REQUESTS_LIMITS=:
+export SCALE_REPLICAS=1
+export SCALE_WORKERS=20
+export ENABLE_RBAC=true
+export SCENARIO=mvp
+export USE_PR_BRANCH=true
+export WAIT_FOR_SEARCH_INDEX=false
+export RHDH_HELM_CHART=redhat-developer-hub
+export AUTH_PROVIDER=keycloak
+export RHDH_HELM_REPO='https://raw.githubusercontent.com/rhdh-bot/openshift-helm-charts/refs/heads/redhat-developer-hub-1.5-148-CI/installation'
```

Comment:
@jhutar the version for 1.5 should be `1.5-178`

Reply:
Thank you, fixed!